charts/aws-efs-csi-driver/values.yaml

```diff
@@ -175,7 +179,7 @@ node:
   affinity:
     nodeAffinity:
       requiredDuringSchedulingIgnoredDuringExecution:
-        nodeSelectorTerms:
+  serms:
```

Comment:
are these changes here necessary?

Reply:
sorry about this. they seem to have come in when I pulled the latest changes to bring the PR up to date. I'll fix it.

Reply:
Thanks, you can leave another comment when it's updated to help us respond quickly

Reply:
should be corrected now, sorry for the trouble